CHANGES.rst

```diff
@@ -243,6 +246,9 @@ astropy.convolution
 astropy.coordinates
 ^^^^^^^^^^^^^^^^^^^
 
+- Fixed ``astropy.coordinates.concatenate`` to also concatenate velocity (i.e.
```

Comment:
(minor) Maybe "to include velocity in the concatenation" or similar?  Right now it sounds like it didn't concatenate *anything*